src/BaseApiAbstract.php

```diff
@@ -23,7 +23,7 @@ abstract class BaseApiAbstract
 
     const CLIENT_LIB_ID_SDK = 'smartling-api-sdk-php';
 
-    const CLIENT_LIB_ID_VERSION = '3.6.7';
+    const CLIENT_LIB_ID_VERSION = '3.7.0';
```

Comment:
As for me it's backward compatible update and could be treated as minor (3.6.8).

Reply:
Sure, it's backward compatible. According to semver first digit is major release (bc is broken), second - minor release (bc is not broken, new feature added), third - patch (bc is not broken, no new feature added, issue(s) is/are fixed). So, it's minor release with new feature added without bc break.